Tests/TuistKitTests/Mappers/Factories/WorkspaceMapperFactoryTests.swift

```diff
@@ -49,6 +49,17 @@ final class WorkspaceMapperFactoryTests: TuistUnitTestCase {
         XCTAssertContainsElementOfType(got, TuistWorkspaceIdentifierMapper.self)
     }
 
+    func test_default_contains_the_tuist_workspace_render_markdown_readme_mapper() {
+        // Given
+        subject = WorkspaceMapperFactory(projectMapper: SequentialProjectMapper(mappers: projectMapperFactory.default()))
+
+        // When
+        let got = subject.default()
+
+        // Then
+        XCTAssertContainsElementOfType(got, TuistWorkspaceRenderMarkdownReadmeMapper.self)
+    }
```

Comment:
You are testing that the mapper is added, but you also need to add a test for the mapper itself, which checks both scenarios (renderMarkdownReadme=false and renderMarkdownReadme=true)